inference-engine/src/legacy_api/src/net_pass.cpp

```diff
@@ -527,6 +527,12 @@ bool unrollTI(CNNLayerPtr cur, ICNNNetwork& net) {
     for (int i = 0; i < first_class.size(); i++) {
         auto& rule = first_class[i];
         auto out_data = ti->outData[rule.from];
+        
+        if (num == 1) {
```

Comment:
Should we also remove Split layer for `num == 1` case?

Reply:
i would recommend that as well.